lms/resources/frontend_app.py

```diff
@@ -0,0 +1,20 @@
+import functools
+
+from lms.resources._js_config import JSConfig
+
+
+class FrontendAppResource:
```

Comment:
Perhaps this should be a base class for the `LTILaunchResource` resource?

Reply:
I don't think we should introduce a `FrontendAppResource`.

It's not actually true that this is the context resource for views that serve frontend apps (as the docstring below states). In fact, in most cases where a frontend app is served `LTILaunchResource` is the context resource. This `FrontendAppResource` is only used by the `oauth2_redirect_error` view.

I think we should just rename this to `OAuth2RedirectErrorResource`